jetty-home/pom.xml

```diff
@@ -170,6 +170,33 @@
               <outputDirectory>${source-assembly-directory}/lib</outputDirectory>
             </configuration>
           </execution>
+          <execution>
+            <id>copy-lib-logging-deps</id>
+            <phase>generate-resources</phase>
+            <goals>
+              <goal>copy-dependencies</goal>
+            </goals>
+            <configuration>
+              <includeGroupIds>org.eclipse.jetty,org.slf4j</includeGroupIds>
+              <includeArtifactIds>jetty-slf4j-impl,slf4j-api,jul-to-slf4j,jcl-over-slf4j,log4j-over-slf4j</includeArtifactIds>
```

Comment:
The `jul-to-slf4j`, `jcl-over-slf4j` and `log4j-over-slf4j` bridge jars should not be included in jetty-home.  They should be downloaded into jetty-base only if their module is enabled.